src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationBaseTest.java

```diff
@@ -32,10 +39,14 @@
 @Slf4j
 public abstract class IntegrationBaseTest {
 
+    public static final String DLQ_PREFIX = "DLQ.";
```

Comment:
If this constant isn't being used, can it be removed please?

Reply:
We are using it in the IntegrationTestsExtension. I just followed the same structure as in the NHAIS because they are also importing some of the variables from IntegrationBaseTest. Do you think it will be better to move this variable just to the IntegrationTestsExtension for now?

Reply:
Would it break functionality to remove it? 

Personally, I think if it's not being used in this file, then it should be removed but if it'll break something, then leave it in.

Reply:
It will break the functionality. However, because it's not used in that class, I agree with you that it should not be there. Here is the alternative solution 620162e6e296c588ea79ac9956f89206c36eb869. Tell me what about it. Thanks!

Reply:
Yeah I prefer it in that class 😄